docs/features/clipboard.rst

```diff
@@ -1,5 +1,10 @@
 Clipboard Integration
 =====================
 
+  - sent to the operating system paste buffer, by ending with a bare ``>``, as
```

Comment:
Should this begin with something like **Command output can be sent ...***?

Reply:
Yes, this was just a stupid cut and paste from the old documentation. New commit pushed with a real clipboard integration page.